diesel/src/mysql/connection/bind.rs

```diff
@@ -70,3 +155,25 @@ fn mysql_type_to_ffi_type(tpe: MysqlType) -> ffi::enum_field_types {
         MysqlType::Blob => MYSQL_TYPE_BLOB,
     }
 }
+
+fn known_buffer_size_for_ffi_type(tpe: ffi::enum_field_types) -> Option<libc::c_ulong> {
+    use std::mem::size_of;
+    use self::ffi::enum_field_types;
```

Comment:
Not for this one. `enum_field_types::MYSQL_TYPE_TINY => stuff()` will run stuff if the value is equal to `MYSQL_TYPE_TINY`. `MYSQL_TYPE_TINY => stuff()` will run stuff for all possible values (and also fail to compile since we have warnings disabled and using a catch-all ident which matches an imported name in a pattern warns)